File: MVCFornecedores/Data/Entities/Fornecedor.cs
using MVCFornecedores.ViewModels;
using System;
using System.Collections.Generic;

namespace MVCFornecedores.Data.Entities
{
    public class Fornecedor
    {
        public int Id { get; set; }

        public string Nome { get; set; }
        public string CPF_CNPJ { get; set; }

        public Empresa Empresa { get; set; }

        public string RG { get; set; }

        public DateTime? DataCadastro { get; set; }

        public DateTime? DataNascimento{ get; set; }

        public ICollection<Contato> Contatos { get; set; }
    }
}

File: MVCFornecedores/client/src/app/model/fornecedor.ts
import { Contato } from './contato';

export class Fornecedor {
    id: number = 0;
    nome: string = "";
    cpF_cnpj: string = "";
    rg: string = "";
    dataCadastro?: Date;
    dataNascimento?: Date;
    contatos?: Contato[] = new Array<Contato>();
}